diagnostic.md

```diff
@@ -57,13 +68,13 @@ Please write a method that traverses from a list item (`li`) DOM node to the
 body DOM node, then hides it.
 
 <!-- your answer starts here -->
-
+$("li").hide("body")
```

Comment:
Yes, but we want to traverse to the `body`. Something like `$('li').parent().parent().hide();`